resources/rest-service/cloudify/migrations/versions/9d261e90b1f3_5_1_1_to_5_1_2.py

```diff
@@ -0,0 +1,134 @@
+"""5_1_1 to 5_1_2
+
+- Add columns state and error to blueprints table
+
+Revision ID: 9d261e90b1f3
+Revises: 5ce2b0cbb6f3
+Create Date: 2020-11-26 14:07:36.053518
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from manager_rest import storage
+
+from cloudify.models_states import VisibilityState
+
+from manager_rest.storage.models_base import JSONString
+
+# revision identifiers, used by Alembic.
+revision = '9d261e90b1f3'
+down_revision = '5ce2b0cbb6f3'
+branch_labels = None
+depends_on = None
+
+VISIBILITY_ENUM = postgresql.ENUM(VisibilityState.PRIVATE,
+                                  VisibilityState.TENANT,
+                                  VisibilityState.GLOBAL,
+                                  name='visibility_states',
+                                  create_type=False)
+
+
+def upgrade():
+    _blueprint_async_upload_changes()
+    _create_filters_table()
+
+
+def downgrade():
+    _revert_blueprint_async_upload_changes()
+    _drop_filters_table()
+
+
+def _blueprint_async_upload_changes():
```

Comment:
not in this pr?

Reply:
Right, @glukhman created this migration file and I used it. Will fix it to include only my functions, and when the time comes, will rebase.